episodes/formative-assessment.md

```diff
@@ -22,6 +22,15 @@ After completing this episode, participants should be able to...
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+![
+In the next two episodes, we will design assessments to measure learners' attainment 
+of the objectives we defined previously.
+](./fig/cldt-step-3.svg){
```

Comment:
Shouldn't this be cldt-step-2.svg diagram?

Reply:
yes, it should. Good catch.